cms/envs/common.py

```diff
@@ -660,8 +661,15 @@
     'openedx.core.djangoapps.header_control.middleware.HeaderControlMiddleware',
     'django.middleware.cache.UpdateCacheMiddleware',
     'django.middleware.common.CommonMiddleware',
+
+    # CORS and CSRF
     'django.middleware.csrf.CsrfViewMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
+    'openedx.core.djangoapps.cors_csrf.middleware.CorsCSRFMiddleware',
+    'openedx.core.djangoapps.cors_csrf.middleware.CsrfCrossDomainCookieMiddleware',
     'django.contrib.sites.middleware.CurrentSiteMiddleware',
```

Comment:
Is the `django.contrib.sites.middleware.CurrentSiteMiddleware` meant to be in the CORS and CSRF section?

Reply:
good catch. Moved that out.